src/styles/stopwatch.scss

```diff
@@ -0,0 +1,69 @@
+.page {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  margin: 0;
+}
```

Comment:
page is not part of stopwatch block [Typical BEM Mistakes](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#styling-a-block-in-the-context-of-another-block)